### ISSUE TITLE
Is this still needed with yamlcpp0.8?

### ISSUE THREAD

**Issue by Ryanf55** [ros2/yaml_cpp_vendor#44]
https://github.com/jbeder/yaml-cpp/releases/tag/0.8.0

I contributed some CMake support to yaml-cpp, and worked with the maintainer to cut a release. Given that it now supports a CMake export in config mode, is it time to retire the vendored version?

**Comment by clalancette** [comment]
We still need the vendored version for Windows, as far as I know.  Also, 0.8.0 is not available in Ubuntu 22.04.

**Comment by Ryanf55** [comment]
> We still need the vendored version for Windows, as far as I know. Also, 0.8.0 is not available in Ubuntu 22.04.

Looks like the debian maintainers are doing a nice job of moving the release 0.8.0 into testing. It might hit 22.04 soon. 
https://tracker.debian.org/pkg/yaml-cpp
```
[2023-08-29] [yaml-cpp 0.8.0+dfsg-3 MIGRATED to testing ](https://tracker.debian.org/news/1458162/yaml-cpp-080dfsg-3-migrated-to-testing/)(Debian testing watch)
[2023-08-23] [Accepted yaml-cpp 0.8.0+dfsg-3 (source) into unstable ](https://tracker.debian.org/news/1456427/accepted-yaml-cpp-080dfsg-3-source-into-unstable/)(Gianfranco Costamagna)
[2023-08-22] [Accepted yaml-cpp 0.8.0+dfsg-2 (source) into experimental ](https://tracker.debian.org/news/1455922/accepted-yaml-cpp-080dfsg-2-source-into-experimental/)(Gianfranco Costamagna)
[2023-08-21] [Accepted yaml-cpp 0.8.0+dfsg-1 (source amd64) into experimental ](https://tracker.debian.org/news/1455611/accepted-yaml-cpp-080dfsg-1-source-amd64-into-experimental/)(Debian FTP Masters) (signed by: Gianfranco Costamagna)
```

I'll keep an eye on it, and if Ubuntu 22 gets updated, perhaps we can revisit to see if Windows still needs the vendor version.
